packages/suite/src/views/wallet/staking/components/Rewards/index.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useEffect } from 'react';
+import styled from 'styled-components';
+import { Button, variables, Icon, Tooltip } from '@trezor/components';
+import { getReasonForDisabledAction, useCardanoStaking } from '@wallet-hooks/useCardanoStaking';
+import { formatNetworkAmount } from '@wallet-utils/accountUtils';
+import { Translation } from '@suite-components/Translation';
+import ActionInProgress from '../ActionInProgress';
+import { Account } from '@suite/types/wallet';
+import {
+    StyledH1,
+    Actions,
+    Heading,
+    StyledCard,
+    Row,
+    ValueSmall,
+    Text,
+    Value,
+    Title,
+    Column,
+} from '../primitives';
+
+const TitleSecond = styled.div`
+    display: flex;
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    font-size: ${variables.FONT_SIZE.NORMAL};
+    margin-top: 30px;
+`;
+
+const Rewards = (props: { account: Account }) => {
+    const {
+        address,
+        rewards,
+        withdraw,
+        calculateFeeAndDeposit,
+        loading,
+        actionAvailable,
+        deviceAvailable,
+        pendingStakeTx,
+    } = useCardanoStaking();
+    const { account } = props;
+
+    useEffect(() => {
+        calculateFeeAndDeposit('withdrawal');
+    }, [calculateFeeAndDeposit]);
+
+    const actionButton = (
+        <Button
+            isLoading={loading}
+            isDisabled={
+                rewards === '0' ||
+                !actionAvailable.status ||
+                !deviceAvailable.status ||
+                !!pendingStakeTx
+            }
+            icon="T2"
+            onClick={() => withdraw()}
+        >
+            <Translation id="TR_STAKING_WITHDRAW" />
+        </Button>
+    );
+
+    const reasonMessageId = getReasonForDisabledAction(actionAvailable?.reason);
+
+    return (
+        <StyledCard>
+            <StyledH1>
+                <Icon icon="CHECK" size={25} />
+                <Heading>
+                    <Translation id="TR_STAKING_REWARDS_TITLE" />
+                </Heading>
+            </StyledH1>
+            <Text>
+                <Translation id="TR_STAKING_REWARDS_DESCRIPTION" />
+            </Text>
+            <Row>
+                <Column>
+                    <Title>
+                        <Translation id="TR_STAKING_STAKE_ADDRESS" />
+                    </Title>
+                    <ValueSmall>{address}</ValueSmall>
```

Comment:
This is not responsive (unable to see the whole address on some screen sizes). 

![image](https://user-images.githubusercontent.com/3729633/154993379-96100cf5-5b61-4b1e-953b-c504ff30923d.png)

Reply:
Another topic is to implement "Discreet mode" - it should be probably added on some places like this address or available rewards?

Reply:
fixed in https://github.com/trezor/trezor-suite/pull/4144/commits/014db6b9e6eff7541044f0af56a59237b645a338